code/model/editableformfields/EditableCheckboxGroupField.php

```diff
@@ -65,4 +65,9 @@ public function getSelectorField(EditableCustomRule $rule, $forOnLoad = false)
 	public function isCheckBoxField() {
 		return true;
 	}
+
+	public function getSelectorFieldOnly()
+    {
+        return "[name='{$this->Name}[]']";
```

Comment:
Adding the `[]` isn't enough; You need to ensure you compare the value of the specific option with the `FieldValue` specified by the user.